README.md

```diff
@@ -15,7 +15,7 @@ The rules can be run manually or automatically by a job which is based on OpenMR
   * The following is a sample insert command.
 
 ```sql
-insert into dataintegrity_rule(rule_name,rule_category,handler_config,handler_classname) values('Questionable date of death','patient','java','org.openmrs.module.dataintegrity.InvalidDateOfDeath');
+insert into dataintegrity_rule(rule_name,rule_category,handler_config,handler_classname) values('Questionable date of death','patient','java','org.openmrs.module.basicexample.dataintegrity.InvalidDateOfDeath');
```

Comment:
This is just an example so no need to change it - I am thinking that you can add a ticket to add this as an in-built rule instead